ccm_web/client/src/utils/redirect.ts

```diff
@@ -0,0 +1,5 @@
+type RedirectRoute = '/' | '/canvas/redirectOAuth'
+
+export default function redirect (route: RedirectRoute): void {
+  location.href = route
+}
```

Comment:
This feels a bit silly to me, but it does allow us to be make changes to the `location` usage or add logging if we want to, makes things a little more readable, and developers have to register new routes beforehand. I dunno, we can remove it if someone doesn't like it.